pythonforandroid/bootstraps/sdl2/build/jni/src/start.c

```diff
@@ -66,6 +66,36 @@ int file_exists(const char *filename) {
   return 0;
 }
 
+/* Helper function to escape all occurances of ' as \', and all
+ * occurances of \n and \r with \\n and \\r for inline string literals.
+ * (Unix paths can contain line breaks, so to be super safe, we need this) */
+char *escape_quotes_of_inline_string(const char* inline_code_arg) {
+  char *result_buf = malloc(strlen(inline_code_arg) * 2 + 1);
```

Comment:
I don't see where this is freed?

Reply:
Good point, I should really add a note to the comment above the function that it returns something that needs to be `free()`'d